pkg/clients/clients.go

```diff
@@ -11,6 +11,7 @@ type InternalDockerClient interface {
 	ConnectContainerToNetworks(containerName string, settings map[string]*network.EndpointSettings) error
 	ConnectContainerToNetworksByNames(containerName string, networkName []string) error
 	StopAndRemoveDockerContainer() error
+	StopAndRemoveDockerContainerByID(containerID string) error
```

Comment:
This is still very long, can you change it to something like "stopandremovebyid" as we are obviously removing docker containers in this file

Reply:
ok